templates/project/docs/.doctor-rst.yaml.twig

```diff
@@ -40,9 +40,3 @@ whitelist:
 {% if custom_doctor_rst_whitelist_part is not empty %}
     {{ custom_doctor_rst_whitelist_part|raw }}
 {% endif %}
-{% elseif custom_doctor_rst_whitelist_part is not empty %}
-
-whitelist:
-    lines:
-    {{ custom_doctor_rst_whitelist_part|raw }}
-{% endif %}
```

Comment:
What do you mean, "Fix conflict"?

Reply:
There is something weird here:
```
if foo
endif
elseif foo
endif
```

Don't know if this is from a conflict, but I don't understand how it could work.

The twig syntax is wrong to me, and even if it wasn't, there will be to whitelist key.

Reply:
It looks wrong indeed. So what you mean is that you are fixing a bad conflict resolution that happened prior to any of your commits.

> there will be to whitelist key.

Had to read that twice but okay, I get it!

Reply:
IMO the elseif needs to be if. Please let me double check it next week.

I am currently on a phone 📱 

I will merge it then

Reply:
Or please create a separate PR for DOCtor file and merge the others right away. 👍🏻

Reply:
@VincentLanglet the c
Code is correct.

The elseif belongs to the first if:

![image](https://user-images.githubusercontent.com/995707/92370108-21561000-f0fa-11ea-9d4b-277fa1e94ef6.jpeg)

Reply:
I revert it.

Damned, so tricky.

Reply:
Thanks 😊